troubleshooting.md

```diff
@@ -315,3 +315,53 @@ Not doing this will cause Podman in the container to detect that temporary files
 This can cause Podman to reset container states and lose track of running containers.
 
 For running containers on the host from inside a container, we also recommend the [Podman remote client](remote_client.md), which only requires a single socket to be mounted into the container.
+
+### 13) Rootless podman build fails EPERM on NFS:
+
+NFS enforces file creation on different UIDs on the server side and does not understand User Namespace.
```

Comment:
"Does not understand User Namespaces, which rootless Podman requires"

Reply:
Also, mention other parallel filesystems here, not just NFS

Reply:
Since I don't know of particular issues on the other FS's, nor even which ones have shown a problem, I kept it kind of generic and added to the end: ` Other parallel file systems such as Lustre and SpectrumScale will have similar difficulties.
`